static/app/components/replays/header/detailsPageBreadcrumbs.tsx

```diff
@@ -42,10 +42,19 @@ function DetailsPageBreadcrumbs({orgSlug, replayRecord}: Props) {
         {
           label: (
             <Fragment>
-              {<BaseBadge displayName={labelTitle} project={project} avatarSize={16} />}
+              {
+                <BaseBadge
+                  displayName={project?.name}
```

Comment:
We use `project.slug` as the display name (instead of `name`). e.g. the project selector dropdown should match this value:

<img width="429" alt="image" src="https://github.com/getsentry/sentry/assets/79684/e51d6ca1-f5b8-43ad-95a0-26c9120ff6cd">